FHEM/14_FLAMINGO.pm

```diff
@@ -123,122 +70,148 @@ FLAMINGO_Define($$)
 	$hash->{STATE} = "Defined";
 
 	my $name= $hash->{NAME};
-
-
-	if(!defined $hash->{IODev} ||!defined $hash->{IODev}{NAME}){	
-		AssignIoPort($hash);
-  	};  
+	my $iodev = $a[3] if($a[3]);
+	$iodev = $modules{FLAMINGO}{defptr}{ioname} if (exists $modules{FLAMINGO}{defptr}{ioname} && not $iodev);
+
+	### Attributes ###
+	if ( $init_done == 1 ) {
+		#$attr{$name}{model}	= "unknown"	if( not defined( $attr{$name}{model} ) );
+		$attr{$name}{room}	= "FLAMINGO";
```

Comment:
Damit werden auch alle bisher gesetzten räume überschrieben oder?

Reply:
Nein das ist nicht der Fall, daher habe ich die $init_done Abfrage drin.

Reply:
Ah. Damit soll eine neu erstellte Definition in den Raum FLAMINGO erscheinen, wenn die Defininition manuell erstellt wird 

Im Fall von Autocreate übernimmt das ja Autocreate bereits.

Reply:
Das hast du richtig erfasst :-)